Makefile

```diff
@@ -1,4 +1,17 @@
-all: docs
+.PHONY: all
+all: docs fmt vet test
 
-docs:
-	./.github/helm-docs.sh
```

Comment:
Just noticed this--did you mean to remove the `docs` script?

Reply:
This was copied from elsewhere and this is correctly removed, but left it in `all` target which is fixed in this [commit](https://github.com/DataDog/helm-charts/pull/1019/commits/29c1af504d6d15c77e1826d1a2c3095a2e994fed). 

`helm-docs.sh` is supposed to be run manually, it's described [here](https://github.com/DataDog/helm-charts/blob/main/CONTRIBUTING.md).